service/geopmdpy/service.py

```diff
@@ -759,6 +759,7 @@ def _write_mode(self, client_pid):
             write_pid = client_sid
         else:
             write_pid = client_pid
+        self.check_client(self._write_pid)
```

Comment:
This should really be in it's own PR.  It has nothing to do with `geopm_context`.

Reply:
I can split it out, but it will be a two line commit (this one and the one below that you commented on).

Reply:
I'm fine to leave this in as long as we have an issue for a future test.

Reply:
https://github.com/geopm/geopm/issues/2062